0000-template.md

```diff
@@ -1,97 +1,126 @@
 
-- What is the impact of not doing this?
+Since our goal is to create a few clusters of CBRS networks (aka neutral host networks) for 3rd party MNOs and MVNOs to offload data traffic, it would make little sense to end up with a few dozen nodes randomly spread out around the US. We suggest to initially pick 2-3 small pilot cities and run the program as an “invitation only” beta. Only miners within those cities will be eligible to mine HNT by offering CBRS coverage. Over time, when the model is proven, we will gradually expand to ever greater number of cities.
 
-# Unresolved Questions
-[unresolved]: #unresolved-questions
+### Specifics of Proof-of-Coverage Model for CBRS LTE/5G
 
-- What parts of the design do you expect to resolve through the HIP process
-  before this gets merged?
+To effectively validate network readiness for CBRS data offload use case, proof-of-coverage should be structured such that it accomplish the following three goals: 
 
-- What parts of the design do you expect to resolve through the implementation
-  of this feature?
+- Validates that the CBRS cell is on and radiating signal 
+- Estimate the usefulness of the location: i.e. is the cell in the woods in Alaska where no phone will ever join it or is in downtown San Francisco with great potential for offloading traffic from multiple MNO networks? 
+- Estimate reliability of backhaul: does the cell have good, reliable backhaul or is it just radiating RF signal with no ability to pass traffic to the Internet 
 
-- What related issues do you consider out of scope for this HIP that could be
-  addressed in the future independently of the solution that comes out of this
-  HIP?
+#### Validating that CBRS Cell is Radiating Signal and Estimating Usefulness of Location 
 
-# Deployment Impact
-[deployment-impact]: #deployment-impact
+Unlike LoRaWan access points, cellular radios do not have the ability to challenge and witness each other in the same way. Cellular networks are architected to facilitate communication exclusively between a base station and a UE (user equipment) with a sim card in it. Base stations themselves do not have a sim card and cannot emulate a UE, which makes it impossible for one base station to “witness” the RF “challenge” issued by another base station.  
 
-Describe how this design will be deployed and any potential imapact it may have on
-current users of this project.
+The good news, however, is that unlike in LoRaWan networks, cellular networks already have a large number of devices that are continuously seeking for a cell tower to attach to. To validate that the CBRS cell is on and, at the same time, to estimate the usefulness of a particular CBRS cell on the network, Helium miner attached to the cell will need to perform the following tasks: 
 
-- How will current users be impacted?
+- Continuously collect data on the unique number of IMSIs attempting to attach to the PLMN(s) broadcasted by the cell. 
```

Comment:
Might need to be clear on how to protect privacy. Depending on juridistiction, IMSIs will be considered personal identifiable information (PII).